lib/etl.js

```diff
@@ -95,8 +104,20 @@ function subscribe(rippled) {
 
     if (data.type === 'validationReceived') {
       data.reporter_public_key = connections[this.url].public_key;
+
+      // Store master key if validation is signed
+      // by a known valid ephemeral key
+      const master_public_key = manifests.getMasterKey(
+        data.validation_public_key);
+      if (master_public_key) {
```

Comment:
So there's either
- validation_public_key (no manifest)
or
- ephemeral_public_key and master_public_key (manifest)

I'm not sure I like calling the validation_public_key (no manifest) the master key.
I prefer to think about it as what is the identifying key for the validator, and that is:
- validation_public_key (no manifests)
- master_public_key (manifests)

So I think those should occupy the same column. We could add an option ephemeral key column, but I don't think it is useful and we can just determine it later by looking at the manifests.